src/main/java/app/coronawarn/verification/service/TanService.java

```diff
@@ -182,12 +172,12 @@ private VerificationTan persistTan(String tan, TanType tanType, TanSourceOfTrust
    */
   public String generateTeleTan() {
     return IntStream.range(0, TELE_TAN_LENGTH)
-        .mapToObj(i -> TELE_TAN_ALLOWED_CHARS.charAt(Holder.NUMBER_GENERATOR.nextInt(TELE_TAN_ALLOWED_CHARS.length())))
-        .collect(Collector.of(
-            StringBuilder::new,
-            StringBuilder::append,
-            StringBuilder::append,
-            StringBuilder::toString));
+      .mapToObj(i -> TELE_TAN_ALLOWED_CHARS.charAt(Holder.NUMBER_GENERATOR.nextInt(TELE_TAN_ALLOWED_CHARS.length())))
+      .collect(Collector.of(
```

Comment:
`Collectors.joining()`?

Reply:
well i think current way also fits needs.

Reply:
I don't dispute that. I just thought it's a bit laborious to create a custom `Collector` for something that there's a prepared one available. Of course, your call.